_utilities/layout-grid.md

```diff
@@ -508,112 +414,249 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
-  }
-}
-{% endhighlight %}
+<table class="usa-table--borderless site-table-responsive site-table-simple">
+    <thead>
+      <tr>
+        <th scope="col" class="tablet:maxw-card-lg display-inline-flex">Utility</th>
+        <th scope="col" class="display-inline-flex">Mixin</th>
+        <th scope="col" class="display-inline-flex">Example</th>
+      </tr>
+    </thead>
+    <tbody class="font-mono-2xs">
+      <tr>
+        <td data-title="Utility" class="tablet:maxw-card-lg">
+          <span>.grid-<code>container</code>
+          </span>
+        </td>
+        <td data-title="Mixin" class="display-inline-flex">
+          <span>
+            grid-<code>container</code>
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex">
+          <span>
+            @include grid-container
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:text-no-wrap tablet:maxw-card-lg display-inline-flex">
+          <span class="text-normal">
+            .grid-<code>row</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex">
+          <span>
+            grid-<code>row</code>
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex">
+          <span>
+            @include grid-row
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:text-no-wrap tablet:maxw-card-lg display-inline-flex">
+          <span class="text-normal">
+            .grid-<code>col</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex">
+          <span>
+            grid-<code>col</code>(<a href="{{ site.baseurl }}/design-tokens/spacing-units/" class="token">units</a>)
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex">
+          <span>
+            @include grid-col(6)
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:text-no-wrap tablet:maxw-card-lg display-inline-flex">
+          <span class="text-normal">
+            .grid-<code>gap</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex">
+          <span>
+            grid-<code>gap</code>(<a href="{{ site.baseurl }}/design-tokens/spacing-units/" class="token">units</a>)
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex">
+          <span>
+            @include grid-gap("lg")
+          </span>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+  {% include utilities/utility-mixin-using.html %}
+</section>  
+
+<section id="advanced-settings" class="padding-top-4">
+  <h2 class="site-h2 margin-y-0">Advanced settings</h2>
+
+  {% include utilities/responsive-variants.html %}
+  No settings available.
+
+  <div>
```

Comment:
Including the `utilities/responsive-variants.html` allows for this entire section to display (sans the "no settings available")

![image](https://github.com/uswds/uswds-site/assets/25211650/4b0a1689-1893-4665-8aba-d44607222216)

I figured there was value in displaying information on utility breakpoints and which breakpoints are turned on by default, but it carries the responsive variant settings section along with it, which the display grid does not have any settings for.

Maybe we could break down `responsive-variants.html` into smaller more modular includes so that I can pull just the breakpoint table and not the responsive settings table?

Reply:
We could also add a conditional if statement to the `responsive-variants` page to only display utility variants if they exist.

```diff
+  {% if page.utilities %}

  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings">USWDS settings configuration</a>. When modifying settings stored as a map, undefined map elements will keep their default values.</p>

  <div class="example border-left-05 border-secondary-light padding-left-105 margin-top-2 margin-bottom-2 bg-white">
    <h4 class="font-lang-2xs margin-top-0 margin-bottom-05">Example</h4>
    <pre class=" font-mono-xs margin-0 padding-0 bg-transparent">
@use "uswds-core" with (
  $background-color-settings: (
    responsive: true
  )
);
    </pre>
  </div>

  <table class="usa-table--borderless site-table-responsive site-table-simple">
    <thead>
      <tr>
        <th scope="col" class="display-inline-flex">Utility variable</th>
        <th scope="col" class="tablet:maxw-card-lg display-inline-flex">Default</th>
      </tr>
    </thead>
    <tbody class="font-mono-2xs">
      {% for utility in page.utilities %}
        <tr>
          <th scope="row" data-title="Utility variable" class="tablet:text-no-wrap display-inline-flex">
            <span class="text-normal">
              ${{ utility.var }}-settings.responsive
            </span>
          </th>
          <td data-title="Default" class="tablet:maxw-card-lg display-inline-flex">
            <span>
              {% if utility.responsive %}<span class="output-true">true</span>{% else %}false{% endif %}
            </span>
          </td>
        </tr>
      {% endfor %}
    </tbody>
  </table>
+ {% endif %}

Reply:
## Suggestion (blocking)

To avoid any confusion we should only pull in the information that we need and is relevant to the user.

I've imported the table and lowered the opacity on content that isn't relevant, but is included from the partial. The difference is pretty significant.

![image](https://github.com/uswds/uswds-site/assets/3385219/7a4a7b9a-d281-4d2e-b36f-1c9a662cd6ee)

We should either:
1. Find a way to generate the table with only the things we need.
2. Leave this partial out and resolve this in another issue (depends on LOE to separate).

Reply:
I added conditional logic mentioned above to leave out the responsive output section if the utility does not offer any settings. There were no additional regressions in the other utility pages that use this include.

I also left the instruction above the table because it adds instructions on how to implement responsive utility classes.

Resolved in [7cfdb75](https://github.com/uswds/uswds-site/pull/2276/commits/7cfdb75a973c214208fb9d31c537573c86b9d351)